src/Client.php

```diff
@@ -141,21 +141,59 @@ public function create(AutotaskObjects\Entity $obj)
         $params = new AutotaskObjects\CreateParam($obj);
         return $this->_call('create', array($params));
     }
+    
+    public function bulkCreate(array $objs)
+    {
+        if (count($objs) > 200) {
+            throw new \Exception('You can only execute a bulk create on a max of 200 objects per request');
+        }
+        $createObjs = null;
+        foreach ($objs as $obj) {
+            if (!$createObjs){
+               $createObjs = new AutotaskObjects\CreateParam($obj);
+            } 
+            else {
+               $createObjs->Entities[] = $obj;
+            }
+        }
+        return $this->_call('create', array($createObjs));
+    }    
 
     public function update(AutotaskObjects\Entity $obj)
     {
         $params = new AutotaskObjects\UpdateParam($obj);
         return $this->_call('update', array($params));
     }
-
-    public function bulkUpdate(array $objs)
-    {
-        $updateObjs = array();
+    
+    public function bulkUpdate(array $objs) {
+	if (count($objs) > 200) {
```

Comment:
Theres weird indents in this function. Either the `if` is 1 space short, or the rest of the function is 1 space too far to the right I think (its probably the if)